### PR TITLE
docs(RoleManager): fix incorrect example

### PR DESCRIPTION
**Pull request by ceilingfans**
**Please describe the changes this PR makes and why it should be merged:**
This PR fixes the example provided in `RoleManger#delete()` where `deleted` is used as a `Role` when it is `undefined`

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/managers/RoleManager.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ class RoleManager extends CachedManager {
    * @example
    * // Delete a role
    * guild.roles.delete('222079219327434752', 'The role needed to go')
-   *   .then(deleted => console.log(`Deleted role ${deleted.name}`))
+   *   .then(() => console.log('Deleted the role'))
```
Comment (imranbarbhuiya) [inline src/managers/RoleManager.js:230]: The role isn't undefined here. It's the role which got deleted.

Comment (vladfrangu) [reply src/managers/RoleManager.js:230]: The manager doesn't return anything, the role class does.

**Review by vladfrangu — APPROVED** [review]

File: src/managers/RoleManager.js (hunk 1 above)
Comment (ImRodry) [reply src/managers/RoleManager.js:230]: The method returns a Promise<void>, u sure?

Comment (imranbarbhuiya) [reply src/managers/RoleManager.js:230]: > The manager doesn't return anything, the role class does.

Oh my bad.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**